requirements.txt

```diff
@@ -15,6 +15,7 @@ contextlib2==0.5.5
 coverage==4.4.1
 cryptography==1.8.1
 csvx==0.1.1459403998
+datrie==0.7.1
```

Comment:
is this required?

Reply:
"datrie can be used under CPython to improve parsing performance (though in almost all cases the improvement is marginal);" https://github.com/html5lib/html5lib-python